documentation/src/components/component.ts

```diff
@@ -309,3 +309,24 @@ class ComponentElement extends RouteComponent {
     }
 }
 customElements.define('docs-component', ComponentElement);
+
+class StyledElement extends HTMLElement {
```

Comment:
The code sample in the Docs are copyable, but not runnable inline, so the definition needs to actually be set here.